scripts/v120_colors.py

```diff
@@ -16,11 +17,11 @@
 
 mesh2 = mesh.transformed(Translation.from_vector([-6, 0, 0]))
 
-facecolors = {k: [random(), random(), random()] for k in mesh.faces()}
-linecolors = {k: [random(), random(), random()] for k in mesh.edges()}
-pointcolors = {k: [random(), random(), random()] for k in mesh.vertices()}
+facecolors = {k: Color(random(), random(), random()) for k in mesh.faces()}
+linecolors = {k: Color(random(), random(), random()) for k in mesh.edges()}
+pointcolors = {k: Color(random(), random(), random()) for k in mesh.vertices()}
 
 viewer.add(mesh, name="mesh1", show_points=True, facecolors=facecolors, linecolors=linecolors, pointcolors=pointcolors)
-viewer.add(mesh2, name="mesh2", show_points=True, facecolor=[1, 0, 0], linecolor=[0, 1, 0], pointcolor=[0, 0, 1])
+viewer.add(mesh2, name="mesh2", show_points=True, facecolor=Color(1, 0, 0), linecolor=Color(0, 1, 0), pointcolor=Color(0, 0, 1))
```

Comment:
not a big fan of the differentiation between `xxxcolor` and `xxxcolors`. what if both are set? in my opinion, `facecolor` should accept either a single color or a dict. through a descriptor this is always converted to a dict in the background. this is how the regular artists work...

Reply:
yep i prefer way too, gonna make this change